default/generated/spring-framework/spring-framework-5.x-to-6.0-data-access.yaml

```diff
@@ -90,3 +90,39 @@
       url: https://eclipse.dev/eclipselink/downloads/previous_releases.php
     - title: 'Spring 6.0 migration guide'
       url: https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-6.0-Release-Notes#removed-apis
+
+- ruleID: spring-framework-5.x-to-6.0-data-access-00030
+  category: mandatory
+  effort: 1
+  labels:
+  - konveyor.io/source=springboot2
+  - konveyor.io/target=springboot3
+  tags: 
+  - springframework.dao
```

Comment:
Here, the correct field is `tag:`, and it would look something like this:
```yaml
tag:
  - Spring Framework
```

You can take a look at other rules to see how it's usually done